documents/TestSuite/util/constant_color.osl

```diff
@@ -0,0 +1,10 @@
+surface 
```

Comment:
Simple constant color shader. Our generated shaders are connected to Cin to render surfaces.
As noted won't work with surface closures. Will be using a different scene file which won't use this constant shader.